Everest03.NET/AppServices/AppService.cs

```diff
@@ -0,0 +1,45 @@
+using Everest03.NET.Service;
+using Everest03.NET.Services;
+using Everest03.NET.Validators;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Everest03.NET.AppServices
+{
+    public class AppService : IAppService
```

Comment:
Nome do serviço está muito genérico, isso é uma grande má prática quando o serviço não é genérico

Dessa forma especificamos para o que/quem serve o serviço
```suggestion
    public class CustomerAppService : ICustomerAppService
```